cli/running/running.go

```diff
@@ -679,7 +679,8 @@ func FillCtx(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx,
 	runningCtx *RunningCtx, args []string) error {
 	var err error
 
-	if len(args) > 1 && cmdCtx.CommandName != "run" && cmdCtx.CommandName != "connect" {
+	if len(args) > 1 && cmdCtx.CommandName != "run" && cmdCtx.CommandName != "connect" &&
+		cmdCtx.CommandName != "add" {
```

Comment:
Since the `FillCtx` one for all `replicaset` commands `roles add` interface is a bit different from other as we provide 2 arguments `<instance_name> <role_name>` instead of just `<isntance_name>`. So we need to check that the command name is not `add` so that the command doesn't return an error.

I agree that this change is hard to support, but I didn't find how to do it more pretty.

Reply:
Could we detect not "add", but "replicaset add"? Please, add a unti-test for the change than.

Reply:
On this step we haven't context about subcommand "replicaset", only "add".